### PR TITLE
Fix scoping

### LINKED ISSUE #795: Phar does not work after Box prefixing
I tested the latest [phar](https://github.com/infection/infection/releases/download/0.14.0/infection.phar) generated for 0.14.0 release and it does not work:

```php
You are running Infection with PCOV enabled.
     ____      ____          __  _
    /  _/___  / __/__  _____/ /_(_)___  ____
    / // __ \/ /_/ _ \/ ___/ __/ / __ \/ __ \
  _/ // / / / __/  __/ /__/ /_/ / /_/ / / / /
 /___/_/ /_/_/  \___/\___/\__/_/\____/_/ /_/

PHP Fatal error:  Uncaught Error: Call to undefined function _HumbugBoxc963874314ce\Safe\sprintf() in phar:///tmp/infection.phar/src/Locator/FileNotFound.php:15
Stack trace:
#0 phar:///tmp/infection.phar/src/Locator/RootsFileOrDirectoryLocator.php(37): _HumbugBoxc963874314ce\Infection\Locator\FileNotFound::fromFileName('infection.json', Array)
#1 phar:///tmp/infection.phar/src/Locator/RootsFileOrDirectoryLocator.php(53): _HumbugBoxc963874314ce\Infection\Locator\RootsFileOrDirectoryLocator->locate('infection.json')
#2 phar:///tmp/infection.phar/src/Locator/RootsFileOrDirectoryLocator.php(41): _HumbugBoxc963874314ce\Infection\Locator\RootsFileOrDirectoryLocator->innerLocateOneOf(Array)
#3 phar:///tmp/infection.phar/src/Command/InfectionCommand.php(166): _HumbugBoxc963874314ce\Infection\Locator\RootsFileOrDirectoryLocator->locateOneOf(Array)
#4 phar:///tmp/infection.phar/src/Command/InfectionCommand.php(72): _HumbugBoxc963874314ce\Infection\Command\InfectionCommand->runConfigurationCommand(Object(_HumbugBoxc963874314ce\Infection\ in phar:///tmp/infection.phar/src/Locator/FileNotFound.php on line 15

Fatal error: Uncaught Error: Call to undefined function _HumbugBoxc963874314ce\Safe\sprintf() in phar:///tmp/infection.phar/src/Locator/FileNotFound.php:15
Stack trace:
#0 phar:///tmp/infection.phar/src/Locator/RootsFileOrDirectoryLocator.php(37): _HumbugBoxc963874314ce\Infection\Locator\FileNotFound::fromFileName('infection.json', Array)
#1 phar:///tmp/infection.phar/src/Locator/RootsFileOrDirectoryLocator.php(53): _HumbugBoxc963874314ce\Infection\Locator\RootsFileOrDirectoryLocator->locate('infection.json')
#2 phar:///tmp/infection.phar/src/Locator/RootsFileOrDirectoryLocator.php(41): _HumbugBoxc963874314ce\Infection\Locator\RootsFileOrDirectoryLocator->innerLocateOneOf(Array)
#3 phar:///tmp/infection.phar/src/Command/InfectionCommand.php(166): _HumbugBoxc963874314ce\Infection\Locator\RootsFileOrDirectoryLocator->locateOneOf(Array)
#4 phar:///tmp/infection.phar/src/Command/InfectionCommand.php(72): _HumbugBoxc963874314ce\Infection\Command\InfectionCommand->runConfigurationCommand(Object(_HumbugBoxc963874314ce\Infection\ in phar:///tmp/infection.phar/src/Locator/FileNotFound.php on line 15
```

However, PHAR file contains this function when I open it in PHPStorm

`/infection.phar/vendor/thecodingmachine/safe/generated/strings.php`

infection  ~:sites:infection  -  :infection phar:vendor:thecodingmachine:safe:generated:strings php 2019-09-22 14-08-09

cc @theofidry

### PR DESCRIPTION
**Pull request by theofidry**
Closes #795

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,12 +3,15 @@
...
 /*.phar
+/.box_dump/
 /.ci/keys.asc
 /.composer/
 /.idea
 /.php_cs.cache
 /.tools/
 /build/
+/clover.xml
+/coverage/
```
Comment (sanmai) [inline .gitignore]: I wonder if these really required. Can we put these into `build`, which is already ignored?

Comment (theofidry) [reply .gitignore]: guess we can, I always get a bit fed up of looking up for the command so I just copy/paste the one from the doc...

Comment (theofidry) [reply .gitignore]: If you don't mind I would prefer to keep it: it's not hurting and indulge a bit of a lazy copy/paste on the doc when you can't work out your command history :)

**Comment by theofidry** [comment]
tests passing

File: .gitignore (hunk 1 above)
Comment (sanmai) [reply .gitignore]: If that's an issue with command history, how about putting these into a [global .gitignore](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)?

Comment (theofidry) [reply .gitignore]: That would be wrong: at least for the `coverage` directory

Comment (sanmai) [reply .gitignore]: Thank you. I do want avoid everyone adding files left after their favorite IDE.

Comment (theofidry) [reply .gitignore]: ? It's not an IDE thing, it's the infection documentation

Comment (sanmai) [reply .gitignore]: I couldn't find a single mention of clover.xml in the docs, at least with the embedded search there... Where should I be looking?

Comment (theofidry) [reply .gitignore]: No idea, looks like I've been using the wrong thing. Also the doc is using `build` as well now so I wonder where this comes from actually

**Pull request merged**